### ISSUE TITLE
history_on_save?

### ISSUE THREAD

**Issue by asarkar** [vishr/local-history#102]
I see various `history_on*` settings, but nothing is mentioned above `history_on_save`. Is it the default behavior to save history when the user explicitly saves the file? Please document it.
```
"history_on_close": true,
"history_on_focus_lost": false,
"history_on_load": true
```

**Comment by erikdemarco** [comment]
yes i agree. its doesnt make sense you have 'history_on_focus_lost' (which is not really important) but no 'history on save' (which is more important)??

and as additional feature i request "history_on_time" (which save automatically based on every x seconds). this is really important for backup purposes

**Comment by dimon222** [comment]
yes default is save on save.
How to achieve it? - history_on_close = false
